### PR TITLE
[Workers AI] Small changes / tweaks

### LINKED ISSUE #14967: Some Workers AI examples use outdated types
### Existing documentation URL(s)

https://developers.cloudflare.com/workers-ai/models/llama-3-8b-instruct/ and many others

### What changes are you suggesting?

Many of the AI examples still use the following for their types:
```ts
export interface Env {
  AI: any;
}
```

This should instead be:
```ts
export interface Env {
  AI: Ai;
}
```

which will result in a much better developer experience.

### PR DESCRIPTION
**Pull request by kodster28**
- Closes #14967
- Adds in example for Text Summarization models (just 1 rn, but already had been playing around with it so made sense to add)
- Adds visible tagging for `lora` models

### CODE REVIEW

**Review by craigsdennis — APPROVED** [review]
Nice work!

**Comment by mchenco** [comment]
could we capitalize it like `LoRA` ? also, might be worthwhile to add a glossary to the Models page, explaining what each pill means - especially as we'll have more in the future (can be a different PR)

File: layouts/_default/model.html [hunk 1]
```diff
@@ -13,16 +13,20 @@
...
-          {{ range .Params.model.properties }} 
+          {{ range .Params.model.properties }}
             {{ if and (eq .property_id "beta") (eq .value "true") }}
               {{/* Cannot use heading-pill shortcode outside of content dir */}}
               <div class="DocsMarkdown--pill DocsMarkdown--pill-beta">Beta</div>
-            {{ end }} 
+            {{ end }}
+            {{ if and (eq .property_id "lora") (eq .value "true") }}
+              {{/* Cannot use heading-pill shortcode outside of content dir */}}
+              <div class="DocsMarkdown--pill DocsMarkdown--pill-deprecated">Lora</div>
```
Comment (kodster28) [inline layouts/_default/model.html]:
```suggestion
              <div class="DocsMarkdown--pill DocsMarkdown--pill-deprecated">LoRA</div>
```

File: layouts/shortcodes/models-grouped-by-task.md [hunk 2]
```diff
@@ -18,11 +18,15 @@ table th:first-of-type {
...
 {{- range $params.model.properties }}
 {{- if and (eq .property_id "beta") (eq .value "true") }}
 {{- $betaFlag = true }}
 {{- end }}
+{{- if and (eq .property_id "lora") (eq .value "true") }}
+{{- $loraFlag = true }}
 {{- end }}
-| [{{ $params.model_display_name }}{{ if $betaFlag }} <div class="DocsMarkdown--pill DocsMarkdown--pill-beta" style="width: max-content">Beta</div>{{ end }}]({{ .RelPermalink }}) | {{ $params.model.description }} |
+{{- end }}
+| [{{ $params.model_display_name }}{{ if $betaFlag }} <span class="DocsMarkdown--pill DocsMarkdown--pill-beta" style="width: max-content">Beta</span>{{ end }}{{ if $loraFlag }} <span class="DocsMarkdown--pill DocsMarkdown--pill-deprecated" style="width: max-content">Lora</span>{{ end }}]({{ .RelPermalink }}) | {{ $params.model.description }} |
```
Comment (kodster28) [inline layouts/shortcodes/models-grouped-by-task.md]:
```suggestion
| [{{ $params.model_display_name }}{{ if $betaFlag }} <span class="DocsMarkdown--pill DocsMarkdown--pill-beta" style="width: max-content">Beta</span>{{ end }}{{ if $loraFlag }} <span class="DocsMarkdown--pill DocsMarkdown--pill-deprecated" style="width: max-content">LoRA</span>{{ end }}]({{ .RelPermalink }}) | {{ $params.model.description }} |
```

File: layouts/partials/models/code-examples/summarization.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+} satisfies ExportedHandler<Env>;
+```
+
+</details>
+
+<details>
+  <summary>curl</summary>
+
+```bash
+curl https://api.cloudflare.com/client/v4/accounts/{cf_account_id}/ai/run/@cf/facebook/bart-large-cnn \
```
Comment (KianNH) [inline layouts/partials/models/code-examples/summarization.md]: Should `@cf/facebook/bart-large-cnn` be `{{ .Params.model.name }}` like the Worker example?

**Review by KianNH — APPROVED** [review]

File: layouts/partials/models/code-examples/summarization.md (hunk 3 above)
Comment (kodster28) [reply layouts/partials/models/code-examples/summarization.md]: Just introducing more bugs for job security

Comment (kodster28) [inline layouts/partials/models/code-examples/summarization.md]:
```suggestion
curl https://api.cloudflare.com/client/v4/accounts/{cf_account_id}/ai/run/{{ .Params.model.name }} \
```

File: layouts/partials/models/code-examples/summarization.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  <summary>Workers - Typescript</summary>
+
+```ts
+export interface Env {
+  AI: Ai;
+}
+
+export default {
+  async fetch(request, env): Promise<Response> {
+
```
Comment (KianNH) [inline layouts/partials/models/code-examples/summarization.md]:
```suggestion
```

**Pull request merged**